settings.py

```diff
@@ -60,7 +60,6 @@
     ('ar', u'العربية'),  # Arabic
     ('Ar-sa', u'Arabic'),  # Arabic Saudi Arabia
     ('zh', u'中文(简体)'),
-    ('ES419', u'Latin Spanish'),
```

Comment:
Nit: It seems removed unexpectedly. please fix it.

Reply:
Also we are not using this in mcka_apros as well

Reply:
please comment it by adding a comment rather than removing it.
so will take care or remove it in the future if required. 

just saying as we sometimes need to look into the code and to know the purpose of removing it during/in-between upgrading the project.